train_linear_model.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+import pandas as pd
+import joblib
+from sklearn import linear_model
+from sklearn.metrics import mean_squared_error, r2_score
+
+from linear_model import X_test
+
+
+def train(X_train, y_train, saved_model):
+    '''
+    train the model using X_train and y_train. return a model 
+    '''
+    X_train = pd.read_csv(X_train)
+    y_train = pd.read_csv(y_train)
+    model = linear_model.LinearRegression()
+    model.fit(X_train, y_train)
+
+    y_pred_train = model.predict(X_train)
+
+    print("Coefficient:", model.coef_)
```

Comment:
you can use f-string:
print(f"Coefficient: {model.coef_:.2f}" 
.2f prints to 2 decimal places